README.md

```diff
@@ -33,3 +33,5 @@ npm install
 ```
 
 You can then run the site using `hugo server`.
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=329970318)
```

Comment:
```suggestion
[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?repo=cncf/glossary)
```
I think we don't have to hide repo_select and would better to show repository name directly.